website/blog/2018-12-21-state-of-react-native-community.md

```diff
@@ -0,0 +1,51 @@
+---
+title: The State the React Native Community in 2018
+author: Lorenzo Sciandra
+authorTitle: Core Maintainer & React Native Developer
+authorURL: https://github.com/kelset
+authorImageURL: https://avatars2.githubusercontent.com/u/16104054?s=460&v=4
+authorTwitter: kelset
+category: announcements
+---
+
+Two years from now, we’ll probably look back at 2018 as a turning point in the React Native development history.
```

Comment:
This sounds a bit off to me. What do you think about "We made a large number of changes about how we develop and communicate about React Native. We hope that in two years from now we can look back and see that 2018 was a turning point for React Native."?

Reply:
I’m not sure I understand what is off there, but ok 👍